iohub/cli/rechunk.py

```diff
@@ -0,0 +1,169 @@
+import copy
+import itertools
+import json
+import logging
+import multiprocessing as mp
+import os
+from functools import partial
+from glob import glob
+from pathlib import Path
+from typing import Literal, Tuple, Union
+
+import numpy as np
+from dask.array import to_zarr
+from natsort import natsorted
+from numpy.typing import NDArray
+from tqdm import tqdm
+from tqdm.contrib.itertools import product
+
+from iohub.ngff import Plate, Position, TransformationMeta, open_ome_zarr
+
+MAX_CHUNK_SIZE = 500e6  # in bytes
```

Comment:
Hi @edyoshikun , I bumped into this because I'm having issues with chunking when copying data from Daxi. 

MAX_CHUNK_SIZE should be a constraint, except for the 2147483647 bytes limit of zarr.
Otherwise, it decreases iohub flexibility.